docs/changes.txt

```diff
@@ -12,6 +12,7 @@
 - Added hash-mode: AES-256-ECB NOKDF (PT = $salt, key = $pass)
 - Added hash-mode: iPhone passcode (UID key + System Keybag)
 - Added hash-mode: MetaMask Wallet
+- Added hash-mode: VMware VMX (PBKDF2-HMAC-SHA1 + AES-256-CBC)
```

Comment:
Needs update due to v6.2.3 release

Reply:
could you add it after the merge? thanks :)